docs/_source/docs/stack_config.rst

```diff
@@ -100,6 +102,47 @@ hooks
 A list of arbitrary shell or Python commands or scripts to run. Find out more
 in the :doc:`hooks` section.
 
+ignore
+~~~~~~
+* Resolvable: No
+* Can be inherited from StackGroup: Yes
+* Inheritance strategy: Overrides parent if set
+
+This configuration should be set with a boolean value of ``True`` or ``False``. By default, this is
+set to ``False`` on all stacks.
+
+``ignore`` determines how the stack should be handled when running ``sceptre launch``. A stack
+marked with ``ignore: True`` will be completely ignored by the launch command. If the stack does NOT
+exist on AWS, it won't be created. If it *DOES* exist, it will neither be updated nor deleted.
+
+You *can* mark a stack with ``ignore: True`` that other non-ignored stacks depend on, but the launch
+will fail if dependent stacks require resources or outputs that don't exist because the stack has not been
+launched. **Therefore, only ignore dependencies of other stacks if you are aware of the risks of
+launch failure.**
+
+This setting can be especially useful when combined with Jinja logic to exclude certain stacks from
+launch based upon conditional Jinja-based template logic.
+
+For Example:
+
+.. code-block:: yaml
+
+   template:
+       path: "my/test/resources.yaml"
+
+   # Configured this way, if the var "use_test_resources" is not true, the stack will not be launched
+   # and instead excluded from the launch. But if "use_test_resources" is true, the stack will be
+   # deployed along with the rest of the resources being deployed.
+   {% if not var.use_test_resources %}
+   ignore: True
+   {% endif %}
+
+
+.. note::
+   The ``ignore`` configuration **only** applies to the **launch** command. You can still run
```

Comment:
This was my thought as well. Back when I called it "launch_action", it was very clear that it was only intended for use in "launch". However simply saying "ignore" and "obsolete" is a log more generic of a name.

My thoughts on this matter are:
1. Personally, I like the idea that "ignore" and "obsolete" are only ever regarded by "launch" and "prune" actions (and I suppose diff as well). It's cleaner to implement that way and you can always override these by explicitly using `create`, `update`, and `delete`, which know nothing about these configurations.
2. If we were to make all commands care about these, it would be a LOT more work and would require implementation likely within SceptrePlan itself of filtering these things out rather than at the specific CLI command level. This is possible, of course, but it would then mean we'd either lose the ability to override with `create`, `update`, and `delete` or we'd need to have special arguments to override `ignore` and `obsolete`. Both of these seem like messier implementations.
3. I agree that the current names of `ignore` and `obsolete` are a bit deceptive, since it would stand to reason that `ignore` would be globally applied (as opposed to `launch_action`, which seems clearly related to `launch` only.
4. I think `ignore` is the more problematic of the two configs for the above reasons. `obsolete` is more easily understood as only applying to pruning actions.
5. Perhaps we could clarify the `ignore` config by renaming it `ignore_launch`? That seems very clear at what it means to me and doesn't seem like it has to be applied more broadly.